src/components/Navbar.vue

```diff
@@ -1,58 +1,44 @@
 <script setup lang="ts">
-    import { ref } from 'vue';
-    // Définition du type pour les Boutons et commandes
-    interface Bouton {
-    titre: string;
-    link: string;
-    perm: string;
-    }
+import { ref } from 'vue';
 
-    // Utilisation de 'ref' pour la réactivité. Vous pouvez aussi utiliser 'reactive' pour des objets complexes.
-    const boutons = ref<Bouton[]>([
-        { titre: "Acceuil", link: "/" , perm: "all" },
-        { titre: "Recherche", link: "/recherche" , perm: "all" },
-        { titre: "Playlist", link: "/playlist" , perm: "all" },
-        { titre: "Abonnements", link: "/abonnements" , perm: "all" },
-        { titre: "Dashboard", link: "/dashboard" , perm: "admin" },
-        { titre: "Gestion comptes", link: "/gestion-comptes" , perm: "admin" }
-    ]);
+interface Bouton {
+titre: string;
+link: string;
+perm: string;
+}
 
- 
+const boutons = ref<Bouton[]>([
+    { titre: "Acceuil", link: "/" , perm: "all" },
+    { titre: "Recherche", link: "/recherche" , perm: "all" },
+    { titre: "Playlist", link: "/playlist" , perm: "all" },
+    { titre: "Abonnements", link: "/abonnements" , perm: "all" },
+    { titre: "Dashboard", link: "/dashboard" , perm: "admin" },
+    { titre: "Gestion comptes", link: "/gestion-comptes" , perm: "admin" }
+]);
 
-    const commandes = ref<Bouton[]>([]);
-    const searchShowed = ref<boolean>(false);
-    const menunotif = ref<boolean>(false);
-    const menucompte = ref<boolean>(false);
-    const logoPath = ref(' /assets/logo.png');
+const searchShowed = ref<boolean>(false);
+const menunotif = ref<boolean>(false);
+const menucompte = ref<boolean>(false);
+const logoPath = ref(' /logo.png');
 
-
-    // Méthode pour ajouter un Bouton aux commandes
-    function commander(bouton: Bouton) {
-        commandes.value.push(bouton);
-    }
 </script>
 
 <template>
     <v-app-bar :elevation="2" rounded>
-        <!-- Contenu à gauche -->
         <img class="mr-3" :src="logoPath" height="40"/>
 
-        
-        <!-- Itération sur chaque bouton pour créer un bouton par bouton -->
             <div v-for="bouton in boutons" :key="bouton.titre">
-                <!-- Correction de la méthode commander appelée avec l'objet bouton -->
-                <v-btn variant="plain" color="bar" @click="commander(bouton)">{{ bouton.titre }}</v-btn>
+                <v-btn variant="plain" color="bar" >{{ bouton.titre }}</v-btn>
             </div>
         
         <v-spacer></v-spacer>
-        <!-- Contenu à droite -->
           
         <v-expand-transition>
-            <v-text-field label="Recherche" v-if="searchShowed" ></v-text-field>
+            <v-text-field hide-details label="Recherche" v-if="searchShowed" ></v-text-field>
```

Comment:
Collapse tag without content
```suggestion
            <v-text-field hide-details label="Recherche" v-if="searchShowed" />
```